src/Google.Api.CommonProtos/Type/MoneyPartial.cs

```diff
@@ -0,0 +1,47 @@
+﻿/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file or at
+ * https://developers.google.com/open-source/licenses/bsd
+ */
+
+using System;
+
+namespace Google.Type
+{
+    public partial class Money
+    {
+        /// <summary>
+        /// The amount of money in <see cref="decimal"/> format.
+        /// </summary>
+        /// <exception cref="ArgumentOutOfRangeException">The integral part of the decimal must be a valid <see cref="Int64"/>, and the fractional part must have a maximum of 9 digits of precision.</exception>
+        public decimal DecimalValue {

```

Comment:
Nit: please use "brace at start of line" style for manual code.
(I know protoc uses "brace at end of line" but that's more historical than anything else.)